### PR TITLE
Added support for naming tokens

### PR DESCRIPTION
**Pull request by tal-codefresh**

### CODE REVIEW

File: lib/interface/cli/commands/auth/token/create-token.cmd.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    create = require('../../root/create.cmd'),
+    createToken = require('../../../../../logic/api/token').createToken;
+
+module.exports = new Command({
+    command: 'token',
+    description: 'Issues a new API token to use with Codefresh\'s CLI',
+    root: true,
+    parent: create,
+    builder(yargs){
+        return yargs.option('name', {
```
Comment (itai-codefresh) [inline lib/interface/cli/commands/auth/token/create-token.cmd.js]: name should be part of the command

Comment (tal-codefresh) [reply lib/interface/cli/commands/auth/token/create-token.cmd.js]: You mean "positional" arg?

Comment (itai-codefresh) [reply lib/interface/cli/commands/auth/token/create-token.cmd.js]: yes

**Comment by tal-codefresh** [comment]
Thanks for your comments!
I'd really like to dig down into your design rationale before moving on any further here.

Let's take this offline: Closing PR for now.

**Pull request closed without merging**